### PR TITLE
schemas.rst update

### PR DESCRIPTION
**Pull request by satendrakumar06**
Removed NotNull and Nullable keyword from doc.

### CODE REVIEW

**Comment by satendrakumar06** [comment]
@cvogt   I am not able to edit index page so please by:
Goto [Index page](http://slick.typesafe.com/doc/3.1.1/genindex.html) and click on edit this page on github

 It is showing 404 Page not found

**Comment by cvogt** [comment]
@satendrakumar06 the index page is auto-generated, so no editing there :).

What is the motivation for this change?

**Comment by satendrakumar06** [comment]
@cvogt ok, Thanks for replay. Actually I have removed NotNull and Nullable keyword from doc so I am trying to change index 
```
 .. index:: null, nullable, NotNull
by
 .. index:: null, nullable, non-nullable
```

Is this make sense ?

**Comment by cvogt** [comment]
Why remove `Nullable` and `NotNull` from the docs?

**Comment by satendrakumar06** [comment]
@cvogt There is no keyword O.NotNull ,O.Nullable  in slick  3.1.0+.
correct ?

**Comment by satendrakumar06** [comment]
@szeiger @cvogt  Should I close this ?

**Comment by d6y** [comment]
This change makes sense to me.  As a sanity check in a project I just checked....

```
[error] db.scala:9: value Nullable is not a member of slick.driver.MySQLDriver.ColumnOptions
[error]   def id = column[Long]("id", O.Nullable, O.AutoInc, O.PrimaryKey)
```

However, @satendrakumar06, I don't think we need the additional words that you added.  A few lines above the documentation says...

> Nullable columns are represented by Option[T]

...so I would suggest removing the extra text you've added.

If you want to do that I can `sbt make-site` and see how it renders.

**Comment by satendrakumar06** [comment]
@d6y yes, You are right. Let me remove extra text.

**Comment by satendrakumar06** [comment]
@d6y Please check it.

File: slick/src/sphinx/schemas.rst [hunk 1]
```diff
@@ -78,8 +78,7 @@ object. The following ones are defined for ``JdbcProfile``:
 
 .. index:: null, nullable, NotNull
 
-  Slick automatically marked Optional column as nullable and non-Optional columnas 
-  non-nullable when creating the DDL statements for the table. 
+   Nullable columns are represented by Option[T]
```
Comment (d6y) [inline slick/src/sphinx/schemas.rst]: Hi @satendrakumar06 - slight mis-communication here.  This text ("Nullable columns...") already exists on line 36-37.  I suggest removing it from line 81, unless there's a good reason to duplicate it here.

**Pull request closed without merging**